src/xrpld/app/misc/WasmVM.cpp

```diff
@@ -57,11 +57,20 @@ runEscrowWasm(
 
     std::int64_t const sgas = gasLimit;  // vm.getGas();
     auto ret = vm.run(wasmCode, funcName, imports);
+
+    // std::cout << "runEscrowWasm, mod size: " << wasmCode.size()
```

Comment:
Please remove the comments or add them as proper debug comments

Reply:
This output shouldn't be present. It is here just for future fixing if some issues appear. I'll leave them just in case. They'll  not be present in PR  for `develop` branch.